addons/io/org.openhab.io.homekit/README.md

```diff
@@ -27,8 +27,80 @@ org.openhab.homekit:thermostatHeatMode=HeatOn
 org.openhab.homekit:thermostatAutoMode=Auto
 org.openhab.homekit:thermostatOffMode=Off
 org.openhab.homekit:networkInterface=192.168.0.6
+```
+
+The following additional settings can be added or edited in PaperUI after switching to expert mode:
 
 ```
+org.openhab.homekit:name=openHAB
+org.openhab.homekit:minimumTemperature=-100
+org.openhab.homekit:maximumTemperature=100
+```
+
+### Overview of all settings
+
+<table>
```

Comment:
@cubistico 

Sorry for dragging it so long, but please use markdown tables instead of HTML.

Reply:
I am sure we filed some PRs for binding readmes.
So it could be possible that we didn't reach to the io folder yet.

To be safe:
Markdown Tables are supported and can be converted easily with https://www.tablesgenerator.com/markdown_tables

Reply:
Thanks, that was helpful!